dbt/tests/generic/test_column_is_subset_of_external_column.sql

```diff
@@ -0,0 +1,28 @@
+-- Test that a given column is a subset of a column in an external relation.
+--
+-- Returns rows where `column` has no matches in the external relation, with
+-- additional return columns drawn from the base relation as specified by
+-- `additional_select_columns`.
+{% test column_is_subset_of_external_column(
+    model,
+    column,
+    external_model,
+    external_column,
+    additional_select_columns=[]
+) %}
+
+    {%- set additional_select_columns_csv = additional_select_columns | join(", ") %}
+    {%- set columns_csv = additional_select_columns_csv ~ ", " ~ column %}
+
+    with
+        distinct_external_values as (
+            select distinct ({{ external_column }}) as external_column
+            from {{ external_model }}
+        )
+    select {{ columns_csv }}
+    from {{ model }}
```

Comment:
**question (non-blocking):** Just to check my understanding, `{{ model }}` here is derived from the yaml `name: pardat` and subset by the `config:` where clause, is that correct?

Reply:
Yes! When you use a `config.where` clause, dbt ends up substituting `{{ model }}` with a subquery like `(select * from {{ model }} where {{ where }}) as dbt_subquery`. Some gory details on this process and ways of customizing it are [here](https://docs.getdbt.com/reference/resource-configs/where#custom-logic). One side effect of this implementation is that you can't add an additional alias to a model if you use a `config.where` clause (i.e. `from {{ model }} as alias` throws an error) which is why we don't namespace the `{{ column }}` reference on line 25 even though I tend to prefer namespacing for explicitness wherever possible.